scripts/svm/simpleFill.ts

```diff
@@ -50,7 +49,7 @@ async function fillV3Relay(): Promise<void> {
   const inputAmount = new BN(resolvedArgv.inputAmount);
   const outputAmount = new BN(resolvedArgv.outputAmount);
   const originChainId = new BN(resolvedArgv.originChainId);
-  const depositId = intToU8Array32(new BN(resolvedArgv.depositId));
+  const depositId = resolvedArgv.depositId;
```

Comment:
why change this

Reply:
Same :) This was a mistake when merging/solving conflicts. Fixed now.